.github/workflows/flutter-format.yml

```diff
@@ -0,0 +1,29 @@
+name: Flutter Format
+on:
+  push:
+    paths:
+      - '.github/workflows/**'
+      - 'client/flutter/**'
+  pull_request:
+    branches:
+      - master
+    paths:
+      - '.github/workflows/**'
+      - 'client/flutter/**'
+
+jobs:
+  test:
+    name: Format on ${{ matrix.os }}
+    runs-on: ${{ matrix.os }}
+    strategy:
+      matrix:
+        os: [ubuntu-latest, macos-latest]
```

Comment:
I don't think we need to run format in both Ubuntu and macOS. The output should be the same, otherwise it is a bug in the Flutter SDK. In case the output differs between OSes, our contributor only has 1 machine so s/he cannot fix it anyway.

We should run the format on Ubuntu only to save resources.

Reply:
Seems right. I thought so too. Do you have any idea why [Flutter Test](https://github.com/WorldHealthOrganization/app/blob/master/.github/workflows/flutter-test.yml) runs on macOS and Ubuntu?

The same reasoning can be applied to that job.

Reply:
No idea 😆

Reply:
Since we had very little time to set everything up we wanted to be absolutely sure it worked on all dev machines, at that time CI resources were also not yet strained.

In the future I agree that it will make sense to reduce the number of platforms we run the flutter test suite on.